JoinColony/colonyCDapp
# Maintenance: Remove the Colony details and Colony advanced pages

## Description

This removes the following pages and the respective routes:

**Colony advanced settings page**
![image](https://github.com/user-attachments/assets/e3ad0030-3339-42f5-8a3f-fb528899dc0f)

**Colony details page**
![image](https://github.com/user-attachments/assets/4c6413f3-8711-471a-b415-a801c829e5a4)

## Testing

1. Visit http://localhost:8081/planex/details and verify that the page does not exist
2. Visit http://localhost:8081/planex/advanced and verify that the page does not exist

## Diffs

**Deletions** ⚰️

Deleted references to the following:
- Colony details page
- Colony advanced settings page
- Colony details route
- Colony advanced settings route

Resolves #2923
